content/1-introduction-2-review.md

```diff
@@ -0,0 +1,636 @@
+Indeed, following the synthesis provided by @lewontin_cultural_1982:
+for any type of item, the combination of (a) a transmission process leading to nonrandom dependence of subsequent instances on preceding instances (e.g. offspring and parent phenotype), (b) at least some variation of the properties of items (against which to select), and (c) some level of differential survival and spread depending on those properties, will lead to evolution by natural selection.
+By looking at genetic models of evolution as a special case of those general principles of evolution, @cavalli-sforza_cultural_1981 developed the mathematical analysis of purely phenotypic transmission, which can take different paths in a given population:
+vertical (from parent to offspring), oblique (from a non-parent member of the previous generation, to member of the next generation), horizontal (inside one generation), or any combination thereof.
+@boyd_culture_1985 [-@boyd_origin_2005] further developed this line, leading to the formulation of *Standard Cultural Evolution* (hereafter SCE) which offers a systematic analysis of part of the interactions between cultural and genetic evolution, an approach which is now vibrant with empirical work [see @acerbi_if_2015 for a review of recent studies].
+A notable feature of this programme is that it does not constrain itself into a particular view of what culture is.
+While the main authors do define culture as "information that people acquire from others by teaching, imitation, and other forms of social learning" [@boyd_origin_2005, p. 3], a definition which at first sight might prove difficult to reconcile with a non-informational view of culture (such as the incorporated views from @sec:intro-review-social), the mathematical models do not constrain the concept of culture as much.
+The approach also has deep links with the analysis of *Niche Construction* [e.g. @odling-smee_niche_2003], which offers promising steps towards a reconciliation with non-informational views of culture stressing the importance of the development process in evolution.
+I return to this subject in more detail in @sec:intro-review-developments.
+
+\add{mesoudi-cultural-2011?}
+
+
+#### Parisian {#sec:intro-review-parisian}
+
+In the mid-nineties Dan Sperber formalised a second influential approach to the question of the evolution of culture:
+in a series of innovative articles gathered in @sperber_explaining_1996, the author puts forward a research programme called *Epidemiology of Representations* (better known today as *Cultural Attraction Theory*, hereafter noted CAT), and seeks to provide the cognitive and social sciences with a common framework with which to address interdisciplinary questions.
+One of the guiding questions of Sperber's work is the following: how can we explain both the diversity of culture across regions, and its relative stability through time, knowing that all human beings are more or less made of the same ingredients?
```

Comment:
Indeed, en te lisant je crois qu'il y a deux questions de fond. Celle que tu rappelles, qui je crois est la question de l'épidémiologie des représentations :

> To explain culture, then, is to explain why and how some ideas happen to be contagious.
(Sperber 1996, p. 1)

> So, to explain culture is to answer the following question: why are some representations more successful in a human population, more 'catching' than others?
(Sperber 1996, p. 58)

> Epidemiology of representations [...] has to explain why some representations remain relatively stable - that is, why some representations become properly cultural
(Sperber 1996, p. 59)

Mais cette question ne fait sens qu'une fois introduite la vision de la culture comme ensemble de représentations en circulation. Au fond, je crois que la démarche de Sperber est : 

1. Il y a une tension entre d'un côté la diversité culturelle, et de l'autre le fait que les humains sont une seule espèce, donc relativement uniformes cognitivement (car uniformes biologiquement). Ça correspond peu ou prou à la tension entre sciences sociales et "sciences psychologiques" (si tu me permets le terme :) )
2. Voir la culture comme distribution de représentations (une inspiration au fond basée sur le fait que les sciences cognitives, et leur notion de représentation, sont si prometteuses) promet de résoudre cette tension : ça donne une ontologie cohérente, et devrait permettre de faire se rejoindre une vue nativiste de l'humain et la reconnaissance de la diversité des cultures : 

> The human mind, Chomsky argued, is better viewed not as general all-purpose intelligence, but as a combination of many devices that are in part genetically programmed. These 'modules' (to use the term made popular by Jerry Fodor) are differendy specialized, in terms of both the cognitive domains they handle and the type of information processing they perform. There is a tension, however, between this rather nativisit view of cognition and the recognition of human cultural diversity, which suggests, to the contrary, that the mind is indefinitely malleable. One way of resolving the tension would be to deny, or downplay, the modularity of the mind. In this chapter I do just the opposite: I argue for massive modularity. I then try to show that strong, genetically determined, cognitive predispositions, not only are quite compatible with the kind of cultural diversity we encounter, but even contribute to the explanation of this diversity.
(Sperber 1996, p. 8)

3. Alors si on s'engage dans une vue épidémiologique de la culture, la question devient celle de l'épidémiologle des représentations, à savoir stabilité macro alors qu'il y a du bruit micro, ei on déroule tout l'arsenal attracteurs, modularité de l'esprit, etc.

Que penses-tu d'une présentation avec ces trois pas ? (Ça recouvre une partie du commentaire ci-dessous aussi)

TODO (modulo approval)
- [ ] fix plurals
- [ ] present Sperber in these 3 steps, then noting that it converged with SCE (but starting from a different concern)